Android build failed, references BeaconBroadcastPackage

I get this build error when I download the ibeacon package. This only happens on the android side. 

`/node_modules/react-native-ibeacon-simulator/android/src/main/java/com/ibeacon/simulator/BeaconBroadcastPackage.java:20: error: method does not override or implement a method from a supertype`

Any thoughts?

--- a/android/src/main/java/com/ibeacon/simulator/BeaconBroadcastPackage.java
+++ b/android/src/main/java/com/ibeacon/simulator/BeaconBroadcastPackage.java
@@ -12,17 +12,14 @@ import com.facebook.react.uimanager.ViewManager;
 import com.facebook.react.bridge.JavaScriptModule;
 
 public class BeaconBroadcastPackage implements ReactPackage {
-    @Override
     public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
       return Arrays.<NativeModule>asList(new BeaconBroadcast(reactContext));
     }
 
-    @Override
     public List<Class<? extends JavaScriptModule>> createJSModules() {
       return Collections.emptyList();
     }
 
-    @Override
     public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
       return Collections.emptyList();
     }

